bin/templatemap_generator.php

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env php
 <?php
 
-$help = <<< EOH
+$help = <<<'EOH'
```

Comment:
Should this script be ported to laminas-cli and documented?
(Good candidate for https://github.com/laminas/laminas.github.io/issues/180)

Reply:
👍 https://github.com/laminas/laminas-view/issues/218